### ISSUE TITLE
Shipping class help text links to old (pre-2.6) taxonomy

### ISSUE THREAD

**Issue by jrfoell** [woothemes/woocommerce#11586]
## EXPLANATION OF THE ISSUE

Help text link to Product Shipping Classes when creating a new flat rate class links to old (pre-2.6) shipping class taxonomy.

## STEPS TO REPRODUCE THE ISSUE

In a new install, navigate to wp-admin.
Proceed to WooCommerce -> Settings -> Shipping Tab.
Click the "Rest of the World" shipping zone.
Click the Add Shipping Method button.
In the dialog, select "Flat Rate" and click the Add Shipping Method button.
Click the "Settings" link under the newly created shipping method.
On the settings screen, under the "Shipping Class Costs" heading, click the link for "product shipping class."
Observe WP error.

## SYSTEM STATUS REPORT

```
`
### WordPress Environment ###

Home URL: https://totalplay.cimbura.mac
Site URL: https://totalplay.cimbura.mac
WC Version: 2.6.4
Log Directory Writable: ✔
WP Version: 4.5.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache/2.4.18 (Unix) PHP/5.6.23 LibreSSL/2.2.7
PHP Version: 5.6.23
PHP Post Max Size: 8 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.49.1
OpenSSL/1.0.2h

SUHOSIN Installed: –
MySQL Version: 5.7.13
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.6.4
: 
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔

### Active Plugins (37) ###

Query Monitor: by John Blackbourn – 2.11.4
404 Error Logger: by Samir Shah – 1.0.2
BackupBuddy: by iThemes – 7.0.3.12
BruteProtect: by Automattic – 2.5
Contact Form 7: by Takayuki Miyoshi – 4.4.2
Debug Bar: by wordpressdotorg – 0.8.4
Disable Emails: by WebAware – 1.2.5
EWWW Image Optimizer Cloud: by Shane Bishop – 2.9.3
Favicon by RealFaviconGenerator: by Philippe Bernard – 1.3.6
GhostMonitor WooCommerce: by Ghostmonitor INC – v1.7.0
Google XML Sitemaps: by Arne Brachhold – 4.0.8
Interactive World Maps: by Carlos Moreira – 1.8.1
InfiniteWP - Client: by Revmakx – 1.6.0
Log Emails: by WebAware – 1.1.0
Really Simple SSL: by Rogier Lankhorst – 2.3.9
Redirection: by John Godley – 2.4.5
SEO Ultimate: by SEO Design Solutions – 7.6.5.8
Sucuri Security - Auditing, Malware Scanner and Hardening: by Sucuri
INC – 1.8.1

Taxonomy Metadata: by mitcho (Michael Yoshitaka Erlewine)
sirzooro – 0.5

Total Playgrounds Lookup Table: by Tom Penney – 0.1.0
Tribe Object Cache: by Peter Chester
Modern Tribe
Inc. – 1.0.4

Which Template File: by Gilles Dumas – 4.2.2
WooCommerce Product Enquiry: by Danyo Borg – 99.1.3
WooCommerce Advanced Notifications: by WooThemes – 1.1.20
WooCommerce Save & Share Cart: by cxThemes – 2.07
WooCommerce Braintree Gateway: by WooThemes / SkyVerge – 3.2.0 – 3.3.2 is available
WooCommerce Google Analytics Integration: by WooThemes – 1.4.0
WooCommerce Product Add-ons: by WooThemes – 2.7.21
WooCommerce Predictive Search PRO: by A3 Revolution – 3.3.1
WooCommerce Simple Product Badge: by Pragmatic Mates s.r.o. – 0.1.0
WooCommerce: by WooThemes – 2.6.4
WooCommerce Helper: by WooCommerce – 1.7.0
Wordfence Security: by Wordfence – 6.1.12
WP Migrate DB Pro Media Files: by Delicious Brains – 1.4.5
WP Migrate DB Pro: by Delicious Brains – 1.6.1
YITH WooCommerce Wishlist: by YITHEMES – 2.0.16
YITH WooCommerce Zoom Magnifier: by YITHEMES – 1.2.20

### Settings ###

Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔

### WC Pages ###

Shop Base: #8 - /shop/
Cart: #9 - /cart/
Checkout: #10 - /checkout/
My Account: #12 - /my-account/

### Taxonomies ###

Product Types: bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
variable (variable)


### Theme ###

Name: Flatsome Child Theme
Version: 2.9.2
Author URL: http:%20//www.uxthemes.com/
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 2.9.2
Parent Theme Author URL: http://www.uxthemes.com/
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/cart/proceed-to-checkout-button.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/content-product.php version 2.5.0 is out of date. The core version is 2.6.1
flatsome/woocommerce/content-product_cat.php version 2.5.2 is out of date. The core version is 2.6.1
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/orderby.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/success.php
flatsome-child/product-searchform.php version - is out of date. The core version is 2.5.0
flatsome/woocommerce/single-product/add-to-cart/grouped.php
flatsome/woocommerce/single-product/add-to-cart/simple.php
flatsome/woocommerce/single-product/add-to-cart/variable.php
flatsome/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php version 2.0.14 is out of date. The core version is 2.6.3
flatsome/woocommerce/single-product/product-thumbnails.php version 2.3.0 is out of date. The core version is 2.6.3
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/additional-information.php
flatsome/woocommerce/single-product/tabs/description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php

: Learn how to update outdated templates

### Braintree (Credit Card) ###

Environment: Production
Tokenization Enabled: ✔
````

**Comment by jrfoell** [comment]
![woocommerce settings shipping total playgrounds wordpress](https://cloud.githubusercontent.com/assets/630830/17309263/a4332a56-5803-11e6-8c35-fa45d648568f.png)

**Comment by mikejolley** [comment]
Link goes to `admin.php?page=wc-settings&tab=shipping&section=classes` which is correct.

**Comment by jrfoell** [comment]
I see... it must have been fixed somewhere between 2.6.4 and 2.7.0-dev.

It's still in 2.6.4, just FYI incase someone else stumbles upon this issue.
